src/main/java/module/series/SeriesPanel.java

```diff
@@ -204,7 +205,7 @@ private void initComponents() {
 		deleteButton.setBackground(ThemeManager.getColor(HOColorName.BUTTON_BG));
 		toolbarPanel.add(deleteButton);
 
-		printButton = new JButton(ThemeManager.getIcon(HOIconName.PRINTER));
+		printButton = new JButton(ImageUtilities.getSvgIcon(HOIconName.PRINTER));
```

Comment:
`ThemeManager.getIcon` handles caching of the icons, could you please make a `ThemeManager.getSvgIcon` method that's cache-enabled?

Reply:
I don't know if it's possible because actually getSvgIcon has dynamic parameters (to change color, opacity, etc ...)
So maybe it's better to rollback GroupTeam icons and just modify the printer and rotate icons, remove dynamic parameters in getSvgIcon and manage the cache for it

Reply:
have you seen remark from @weisJ   https://github.com/akasolace/HO/issues/572#issuecomment-645492736

Reply:
> I don't know if it's possible because actually getSvgIcon has dynamic parameters (to change color, opacity, etc ...)

Ok.  The way I handled that for the jerseys was to build the key based on params:
https://github.com/akasolace/HO/blob/master/src/main/java/core/gui/theme/ImageUtilities.java#L540

I don't know if that helps...

Reply:
Thanks, I will have a look this week-end if I have the time, more probably next week and I will do a new PR.